hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.h

```diff
@@ -46,6 +46,23 @@ class HologramDataSourceAvailabilityFetcher: public HologramDataFetcher {
     // to wrong file or malformed file.
     void AcquireConfig(const std::string& config_file_path);
     
+    // Fetches old protos from database if they exist so the program does not
```

Comment:
nit: Fetches availability info from database. 
For comments, no need to expose implementation details.